android-json-form-wizard/src/main/java/com/vijay/jsonwizard/fragments/JsonFormFragment.java

```diff
@@ -81,9 +81,21 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         View rootView = inflater.inflate(R.layout.native_json_form_fragment, null);
         mMainView = (LinearLayout) rootView.findViewById(R.id.main_layout);
         mScrollView = (ScrollView) rootView.findViewById(R.id.scroll_view);
+        setupToolbarBackButton();
         return rootView;
     }
 
+    private void setupToolbarBackButton() {
```

Comment:
Should this be protected in case you want to define your own icon later?

Reply:
Allowed host application to override drawable resource